communication/moderation.md

```diff
@@ -0,0 +1,48 @@
+# Moderation on Kubernetes Communications Channels
+
+This page describes the rules and best practices for people chosen to moderate Kubernetes communications channels, such as Slack and the mailing lists.
+
+## Roles and Responsibilities
+
+As part of volunteering to become a moderator you are now representative of the Kubernetes community and it is your responsibility to remain aware of your contributions in this space. 
+These responsibilities apply to all Kubernetes official channels. 
+
+Moderators _MUST_:   
+
+- Take action as specified by these Kubernetes Moderator Guidelines.
+- Agree to abide by the documented tasks and actions required of moderators.
+- Ensure that the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) is in effect on all official Kubernetes communication channels. 
+- Take care of spam as soon as possible, which may mean taking action by  removing a member from that resource.
+- Fostering a safe and productive environment by being aware of potential multiple cultural differences between Kubernetes community members.
+- Understand that you might be contacted by moderators, community managers, and other users via private email. 
+- Report egregious behavior to steering@k8s.io.
+
+Moderators _SHOULD_: 
+
+- Understand that compassion and empathy is a trait I should display when communicating and collaborating with other community members.
+- Be an example and role model to others in the community.
+- Remember to check and recognize if you need take a break when you become frustrated or find yourself in a heated debate.
+- Help your colleagues if you recognize them in one of the [stages of burnout](https://opensource.com/business/15/12/avoid-burnout-live-happy).
+- Be helpful and have fun! 
+
+## Specific Guidelines
+
+These guidelines are for tool-specific policies that don't fit under a general umbrella. 
+
+### Mailing Lists
+
+
+### Slack
+
```

Comment:
want to add the guidelines link here for now or include all of the specific language that's in here?
[guidelines](https://git.k8s.io/community/communication/slack-guidelines.md)